src/App.js

```diff
@@ -45,52 +87,47 @@ class Image extends React.Component {
                     return response.json();
                 })
                 .then((arrayOfData) => {
-                    this.setState(prevState => ({
-                        images: prevState.images.concat(arrayOfData),
-                        isFetching: false
-                    }));
+                    this.setState((prevState) => {
+                        const buffArray = prevState.images.concat(arrayOfData)
+                        return {
+                            count: buffArray.length,
+                            images: buffArray,
+                            isFetching: false
+                        }
+                    });
                 });
         }
     }
 
-
     render() {
+        const divStyle = {
+            position: 'sticky',
+            top: '20px',
+            marginLeft: '400px',
+            width: '200px',
+        };
+
+        const handleCountChange = (countToDelete) => {
+            this.setState({countToDelete: countToDelete});
```

Comment:
_minor_: you may omit the object value in case the name of the variable equals to the name of the object key :
```js
this.setState({ countToDelete });
```